pennylane/ops/qubit/arithmetic_ops.py

```diff
@@ -210,11 +232,33 @@ def _matrix(cls, *params):
         return QubitSum._mat
 
     @staticmethod
-    def decomposition(wires):
-        decomp_ops = [
+    def compute_decomposition(wires):
+        """Determine ``QubitSum``'s decomposition for specified parameters, wires,
+        and hyperparameters. The decomposition defines an Operator as a product of
+        more fundamental gates:
+
+        .. math:: O = O_1 O_2 \dots O_n.
+
+        ``compute_decomposition`` is a static method and can provide the decomposition of a given
+        operator without creating a specific instance. The instance method ``decomposition`` uses
+        this method and the instance's variables.
+
+        Args:
+            wires (Iterable[Any], Wires): Wires that the operator acts on.
```

Comment:
Again a minor one, but `Wires` is an Iterable, so I usually just use `Iterable` in the type hints to make it easier